virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -40,3 +44,70 @@ def create_disk(disk_type, path=None, size="500M", disk_format="raw", extra='',
     else:
         return libvirt.create_local_disk(disk_type, path=path, size=size,
                                          disk_format=disk_format, extra=extra)
+
+
+def create_custom_metadata_disk(disk_path, disk_format,
+                                disk_device, device_target, device_bus, max_size):
+    """
+    Create another disk for a given path,customize driver metadata attribute
+
+    :param disk_path: the path of disk
+    :param disk_format: the format to disk image
+    :param disk_device: the disk device type
+    :param device_target: the target of disk
+    :param device_bus: device bus
+    :param max_size: metadata_cache max size
+    :return: disk object if created successfully
+    """
+    custom_disk = Disk(type_name='file')
```

Comment:
Here I suggest we use a parameter to pass the Disk(or its subclass) object instead of hard coding Disk(). If that, the codes will have stronger adaptability.  For example, in https://github.com/autotest/tp-libvirt/pull/3350 around line 88, you can reuse this function.

Reply:
updated